README.md

```
@@ -105,3 +105,9 @@ Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To u
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
 -->
+
+## Documentation
+
+Detailed technical documentation for this service is available on DeepWiki:
+
+- https://deepwiki.com/PSMRI/ECD-UI
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)